### PR TITLE
ci: enable deployment of docker images

### PR DESCRIPTION
**Pull request by DanySK**

### CODE REVIEW

File: release.config.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+config.plugins.push(
+    [
+        "@semantic-release/exec",
+        {
+            "publishCmd": publishCmd,
+        }
+    ],
+    "@semantic-release/github",
+    "@semantic-release/git",
+)
```
[the remark this answers is not in the document]
Comment (DanySK) [reply release.config.js:17]: @sonatype-lift ignore

**Comment by eekkelund** [comment]
Container seems to work? Or at least CI manages to build it?

**Comment by DanySK** [comment]
If does build in CI, and I am releasing the image on dockerhub with minor
changes with respect to your contribution (tests and precache).

I automated the release process, that's why you see the message: it's a bot
informing us that the PR has been released after merge.

**Comment by eekkelund** [comment]
Nice!

Yeah, I got it. I was just interested if you managed to test it yourself, with or without docker-_compose_.

**Comment by DanySK** [comment]
I'm running the simulations directly in the building phase of the image. I gladly review and I'm open to further contributions, of course.

Thanks!

**Pull request merged**